relayer/provider/cosmos/provider.go

```diff
@@ -496,10 +497,12 @@ func (cp *CosmosProvider) SendMessages(msgs []provider.RelayerMessage) (*provide
 		return nil, false, err
 	}
 
-	// TODO helper/wrapper function for TxResponse->RelayerTxResponse
+	// helper/wrapper function for TxResponse->RelayerTxResponse?
```

Comment:
Yes I think we want this wrapper function

Reply:
I'll get that added in here!